plugins/aep-consent-status/plugin.test.ts

```diff
@@ -136,7 +136,7 @@ describe('Update Collect Consent', () => {
   it('collect consent is pending', () => {
     const result = plugin([collectConsentP]);
     expect(result).toMatchObject({
-      message: 'Collect consent level is set to pending. Events are queued until the status is updated to yes (events are sent) or no (events are dropped). To update the consent status, check the default collect consent setting or use the update API from the Consent extension and pass in the preferred collect consent settings. Follow the link for more details and code samples.',
+      message: 'Collect consent level is set to pending. Events will be queued until the status is updated to yes (events will be sent) or no (events will be dropped). To update the consent status, check the default collect consent setting or use the update API from the Consent extension and pass in the preferred collect consent settings. Follow the link for more details and code samples.',
```

Comment:
I prefer the present tense for the consent sentences, since it indicates the settings are already into effect, whereas "will be ..." indicates they will be applied sometime in the future which can create some ambiguity. Any strong reason why we are changing these?

Reply:
As discussed, we keep them as the future tense for these.

Reply:
Sounds good. For these changes to take effect I think you also need to bump the version for the consent category